karaf/features/src/main/feature/feature.xml

```diff
@@ -276,60 +276,6 @@
         <feature>jclouds-api-filesystem</feature>
 
         <bundle start-level="85">mvn:org.apache.brooklyn/brooklyn-locations-jclouds/${project.version}</bundle>
-        <feature>brooklyn-software-winrm</feature>
```

Comment:
I had meant to move it to brooklyn-library and then realised it didn't belong there, must have missed putting it back here.  Will fix.